src/transformers/modeling_utils.py

```diff
@@ -28,7 +28,7 @@
 
 import torch
 from packaging import version
-from torch import Tensor, device, nn
```

Comment:
This import was dangerous with so many local variables named `device`. It was only used in a type annotation.